src/onegov/agency/forms/agency.py

```diff
@@ -33,6 +36,28 @@ class ExtendedAgencyForm(Form):
         render_kw={'rows': 10}
     )
 
+    location_address = TextAreaField(
+        label=_("Location address"),
+        render_kw={'rows': 1},
+    )
+    location_code_city = StringField(_("Location Code and City"))
```

Comment:
Please keep format within files consistent (using keyword label everywhere).

Reply:
Not sure what you exactly mean...

Reply:
This is just a cosmetic (and more a general hint to try to keep things consistent within files/modules): Use e.g. `label=_("Location Code and City")`, if a few lines above labels are added using the `label` Argument.

Reply:
Clear!